Update btn and table handling

kevinh-csalabs opened the pull request:
-Updates btn to prevent wrapping of icons while still allowing wrapping of text.
-Updates tables to better size to viewport and stay within containers instead of overflowing external borders
-Standardizes comment style

kevinh-csalabs commented on app/assets/stylesheets/rad_common/rad_bootstrap_overrides.scss:
```diff
@@ -1,18 +1,27 @@
 html {
-  // Bootstrap 4 increased the default font size,
-  // so this just brings it back down so everything is closer to what it was.
+  /* Bootstrap 4 increased the default font size,
```
Standards seem to dictate otherwise... no problem changing if you want to do your own thing.

https://developer.mozilla.org/en-US/docs/Web/CSS/Comments
https://www.w3schools.com/css/css_comments.asp

garyfoster replied:
@kevinh-csalabs we are not using hound for stylesheets nor JS, it's not configured the way we like it - just ignore for now

kevinh-csalabs commented on app/assets/stylesheets/rad_common/rad_bootstrap_overrides.scss:
```diff
@@ -1,18 +1,27 @@
-  white-space: nowrap;
+/* Prevent button icons from wrapping while allowing everything else to wrap */
+.btn {
+	display: inline-flex;
```
hah the old spaces vs tabs. so be it.

kevinh-csalabs commented:
Example of overflow issue in table (table exceed container border):
![Capture](https://user-images.githubusercontent.com/72226237/149839972-ab606ff2-597a-4b48-9b9e-7628542661a6.PNG)

Fixed table overflow (stays within boundary):
![Capture2](https://user-images.githubusercontent.com/72226237/149840511-5f3dc634-c77f-4631-95e9-35fd3561e6f0.PNG)


Example of button overflow issue:
![buttons-overflow](https://user-images.githubusercontent.com/72226237/149840261-991a9f41-1c42-44b0-aa4c-14f03b324cb3.PNG)

fixed button overflow:
![proper-wrapping-buttong](https://user-images.githubusercontent.com/72226237/149840297-409a2729-3022-4d9a-bf2a-076b9f18242e.PNG)

elbzero commented:
LGTM

garyfoster commented:
I'm reverting this, it is regressing on tables, see attached example.
<img width="1592" alt="Screen Shot 2022-01-19 at 5 11 01 PM" src="https://user-images.githubusercontent.com/1126133/150238841-6d96b7c7-2025-4a1f-aaf3-878a3d0b7e72.png">

kevinh-csalabs commented:
@garyfoster ok I'll take a look. To make sure I'm addressing this properly, is the primary concern the extra border or the content fit of the table (table isn't forced to be 100% container width because it causes stretch)?
